package.json

```diff
@@ -59,6 +59,7 @@
     "babel-loader": "^9.2.1",
     "eslint": "^8.57.1",
     "glob": "^8.1.0",
+    "jsdom": "^26.0.0",
```

Comment:
had to move `jsdom` to the root since it's a sibling dependency of `vitest` which is also at the root. otherwise it may not be able to find it as it imports it internally.

also had to override `nwsapi` because in later versions it breaks query selecting for React `useId()` generations due to IDs returning double colons: `:r0:`